src/components/Hub/HubImagesListPreview.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import ImageCard from "./ImageCard";
+import styled from "@emotion/styled";
+
+const SubTitle = styled.span`
+  font-weight: 500;
+  font-size: 1.5rem;
+`;
+
+const HubImagesListPreview = () => {
+  // Todo: update images after fetching it from hub API
+  const images = [
```

Comment:
It would make sense to introduce a type for Images

Reply:
I was deferring this to the next pull request where I actually fetch images from backend.